### PR TITLE
Add specific 2.0 version to helm command when installing the chart

### LINKED ISSUE #1502: Document how to use specific version of the helm chart
Backport it to `v1.3` and `v2.0`

### PR DESCRIPTION
**Pull request by pmalek-sumo**
Fixes #1502

### CODE REVIEW

File: deploy/docs/Installation_with_Helm.md [hunk 1]
```diff
@@ -104,6 +104,7 @@ The following command will install the Sumo Logic chart with the release name
 
 ```bash
 helm upgrade --install my-release sumologic/sumologic \
+  --version=2.0 \
```
Comment (astencel-sumo) [inline deploy/docs/Installation_with_Helm.md:107]: Should this change live in `main` branch or only  in `release-v2.0` branch?

Setting this on `main` means we need to update it when we release `v2.1` and so forth.

Comment (frankreno) [reply deploy/docs/Installation_with_Helm.md:107]: Will this install the latest 2.0 every time? WE will need to keep this updated with every release right?

**Comment by frankreno** [comment]
We will also need to do this in the non-helm install in the k8s tools command.

Rather than adding `--version` to every permutation, would it be better to simply add a new section after 

```If the namespace does not exist, you can add the `--create-namespace` flag.```

That shows a single use of the version flag? The intention of the issue was just to show users who dont know helm how to install specific versions.

**Comment by pmalek-sumo** [comment]
This was intended to be targeted for `release-v2.0` branch so no updates would need to be done to those commands anymore, at this branch.

**Comment by pmalek-sumo** [comment]
Superseeded by https://github.com/SumoLogic/sumologic-kubernetes-collection/pull/1529

**Pull request closed without merging**